plugins/Network/plugin.py

```diff
@@ -306,6 +306,17 @@ def capabilities(self, irc, msg, args, otherIrc):
         irc.reply(format("%L", sorted(otherIrc.state.capabilities_ls)))
     capabilities = wrap(capabilities, ['networkIrc'])
 
+    def saslauth(self, irc, msg, args):
+        """takes no arguments
+
+        Manually initiate SASL authentication.
+        """
+        if 'sasl' in irc.state.capabilities_ack:
+            irc.startSasl(msg)
+            irc.replySuccess()
+        else:
+            irc.error(_('SASL not supported'))
+    saslauth = wrap(saslauth)
```

Comment:
```suggestion
    def authenticate(self, irc, msg, args):
        """takes no arguments

        Manually initiate SASL authentication.
        """
        if 'sasl' in irc.state.capabilities_ack:
            irc.startSasl(msg)
            irc.replySuccess()
        else:
            irc.error(_('SASL not supported'))
    authenticate = wrap(authenticate)
```

we avoid concatenating words in command names; so it has to be either a single a single word or space-separated. I don't have a strong opinion on which is best here.